CONTRIBUTING.md

```diff
@@ -245,3 +246,60 @@ However, these options can lengthen your testing cycle and make the process of c
 ```{bash}
 ./test.sh
 ```
+
+## Releasing to CRAN <a name="cran"></a>
+
```

Comment:
Should we have a note here that this is primarily the responsibility of the maintainer?

Reply:
I added this already:

> This is the exclusively the responsibility of the package maintainer, but is documented here for our own reference and to reflect the consensus reached between the maintainer and other contributors.

I really don't think we have to beat this over the head, since CRAN will prevent anyone other than Brian from pushing a new version anyway.

But I also don't care super strongly, if you ask me to add the parenthetical too I will.

Reply:
I think it's worth it to make it obvious, because it's a sizable blob of text, and it might look intimidating to someone who is trying to learn to contribute.

Reply:
ahhhhh ok, good point! I hadn't thought of that. I thought you and @bburns632 were coming at this from the perspective of "I don't want anyone to accidentally overwrite the package on CRAN", which I wasn't worried about.

This makes sense. Will update now.